CONTRIBUTING-WIP.md

```diff
@@ -36,6 +36,7 @@ Each cdnjs library has a `package.json` file. This file contains required and so
 ### 2.3 Auto-update example
 
 ```js
+  "filename": "function-plot.min.js",
```

Comment:
Even there is no auto-update config, package.json still need to add `filename` this field.
And actually, we mention this field in https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson
> Note that you should have a filename to point to the main file of a lib, if that file is not minified, please still use filename.min.js or filename.min.css structure naming, we'll do the minify job.

But it's good idea that maybe we have to mention about every field in `package.json`

But in this `2.3 Auto-update example` we need to remove `"filename": "function-plot.min.js",`
Because `filename` field is for every lib's package.json, not just for auto-update config.
Many thanks.